src/main/java/org/alfresco/rest/api/impl/SiteMembershipRequestsImpl.java

```diff
@@ -479,5 +511,163 @@ public SiteInfo doWork() throws Exception
 			siteMembershipRequests = siteMembershipRequests.subList(skipCount, end);
 			return CollectionWithPagingInfo.asPaged(paging, siteMembershipRequests, hasMoreItems, totalItems);
 		}
+    }
+
+    @Override
+	public CollectionWithPagingInfo<SiteMembershipRequest> getPagedSiteMembershipRequests(String personId, Paging paging)
+	{
+    	personId = people.validatePerson(personId, true);

```

Comment:
Formatting on this line could be corrected.

Reply:
Class had wrong formatting before changes, only new changes were formatted, I will update.